src/code_80057C60.c

```diff
@@ -885,7 +885,7 @@ void func_800598D4(s32 arg0) {
 }
 
 void func_8005994C(void) {
-    D_8018D214 = 1;
+    D_8018D214 = TRUE;
```

Comment:
Be a little careful with this. It's possible it's a number not a bool. But I'm presuming you checked.

Reply:
I see this variable have only two value who are 0 or 1 and are use in if so I assume it's a bool